go/chat/sender.go

```diff
@@ -437,6 +438,44 @@ func (s *BlockingSender) checkTopicNameAndGetState(ctx context.Context, msg chat
 	return topicNameState, nil
 }
 
+func (s *BlockingSender) resolveOutboxIDEdit(ctx context.Context, uid gregor1.UID,
+	convID chat1.ConversationID, msg *chat1.MessagePlaintext) error {
+	if msg.SupersedesOutboxID == nil {
+		return nil
+	}
+	s.Debug(ctx, "resolveOutboxIDEdit: resolving edit: outboxID: %s", msg.SupersedesOutboxID)
+	typ, err := msg.MessageBody.MessageType()
```

Comment:
small nit: can you add a `MessageBody.IsEdit()` method in extras? just cleaner to read, maybe avdl should just generate this..

Reply:
Sure

Reply:
Actually I kind of like having two different kinds of error here...

Reply:
ok we can leave as is